### ISSUE TITLE
Fix contrast issues 

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/build-an-atom#248]
Currently a proton and neutron can't be distinguished from one another in grayscale. 
During today's design meeting we decided that we would try to solve this issue by first starting with a thicker stroke around the neutron. 

<img width="650" alt="Image" src="https://github.com/user-attachments/assets/9061a0b8-1e5c-4558-90a2-8c602859e538" />

**Comment by Nancy-Salpepi** [comment]
@jbphet was going to try an alternative to the transparent black background that we used for the bucket labels. So, I am going to assign him to this issue as well.

**Comment by jbphet** [comment]
I made the changes we discussed at a recent standup meeting where the background of the button labels have now been removed and the gradients on the buckets have been adjust to improve contrast.  From what I recall of the discussion, the main problem was with the proton bucket, and our target was to get a contrast greater than 3 to 1.  The changes I made put it at 3.52:1 when measured using https://webaim.org/resources/contrastchecker/.  Here are some screenshots:

Current published version (1.6.34):

<img width="757" height="263" alt="Image" src="https://github.com/user-attachments/assets/be0d392b-9248-4e3b-8641-c7fca3eddd63" />

Current main version:

<img width="743" height="265" alt="Image" src="https://github.com/user-attachments/assets/662cff74-0379-4f6b-8e65-b9d3e1257049" />

Contrast measurement for left side of proton label on main version:

<img width="726" height="498" alt="Image" src="https://github.com/user-attachments/assets/3714327b-ca37-4091-bad7-3c413d4ada5d" />

@Nancy-Salpepi - Please review and let us know if we're good to go on this aspect of the contrast improvements.

**Comment by Nancy-Salpepi** [comment]
@AgustinVallejo 
Can you make these changes on the game screen as well?:

-Make the reddish text darker --hex value D14600

- [ ] Element name
- [ ] positive ion, negative ion
- [ ] atomic number
- [ ] charge

-Also, if this is easy, can you 

- [ ] remove the shadow behind the +/- on the net charge meter 
- [ ] add a black stroke around both of them

**Comment by AgustinVallejo** [comment]
Increased. Also feel free to suggest other changes for the X icon, as I made it optionized and it will not change other sims that have this component.

**Comment by Nancy-Salpepi** [comment]
Looks good on this try.
Going to close. We have bigger fish to fry.

**Comment by Nancy-Salpepi** [comment]
@AgustinVallejo can you please 

- [ ] add a white background behind the element name with 70% opacity. 
- [ ] Change the proton border to be the proton base color that is in the color editor D14600.

**Comment by Nancy-Salpepi** [comment]
>I could make the text to shift up as the atom grows (in fact, that's what the code says but it's not doing it for some reason, neither is published), should I do that?

Yes. Let's give that try. Thank you!

**Comment by Nancy-Salpepi** [comment]
also it looks like the white rectangle is in front of the text not behind it

**Comment by Nancy-Salpepi** [comment]
When there aren't a lot of electrons added the top gets very cut off and it reminds me of a fishbowl. Wondering if we can start with the name higher to avoid this as much as possible?

<img width="101" height="103" alt="Image" src="https://github.com/user-attachments/assets/f4f95de8-3251-478a-ac5a-47477cd54a45" />  <img width="160" height="158" alt="Image" src="https://github.com/user-attachments/assets/43b7d174-e6c9-4ccf-a13f-a5f28bd92180" />

**Comment by AgustinVallejo** [comment]
<img width="646" height="483" alt="Image" src="https://github.com/user-attachments/assets/14728a95-2925-45b1-934e-79387a0b1ce1" />
I don't think there's much space to move, as you can see in the side by side comparison. If we move up a small ammount, we'll still have a fishbowl, and moving any more will result in overlap with the orbital.

I can continue reducing the opacity. It was currently at 65%, lowered it to 50% and can keep going down as needed.
